### PR TITLE
Add visualization team and a release repository.

### PR DESCRIPTION
**Pull request by nuclearsandwich**

### CODE REVIEW

File: 00-members.tf
Comment (clalancette) [inline 00-members.tf:187]: For what it is worth, I'm not a huge fan of calling this team `visualization_team`.  There are a lot of repositories in https://github.com/ros-visualization, and the members of this team aren't necessarily the maintainers of all of these repositories.

**Review by clalancette — APPROVED** [review]
I left a comment inline, but I'll still approve and leave it to @nuclearsandwich whether to change it or not.

File: 00-members.tf
Comment (nuclearsandwich) [reply 00-members.tf:187]: @clalancette any suggestion of another name would have been acceptable to me. In the absence of one, I'm going to merge as is but these team names don't propagate out of here so it's perfectly fine to change them later.

**Comment by nuclearsandwich** [comment]
The mirrored repository was manually imported before terraform deployment with
```
bin/import-repo interactive_marker_twist_server-release
```

**Comment by nuclearsandwich** [comment]
This change has been deployed!

File: 00-members.tf
Comment (clalancette) [reply 00-members.tf:187]: OK, that's fair.  I can open a PR to rename this.

**Pull request merged**
